routes/accounts/weekly_summary.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+const express = require( 'express' )
+const router = express.Router()
+const moment = require( 'moment' )
+const sequelize = require( 'sequelize' )
+
+
+const numberOfTasksCreated = ( Item, $gte, user_id ) =>
```

Comment:
Let's move these functions into a separate module that can be imported into this router module.